sway/input/keyboard.c

```diff
@@ -178,18 +178,42 @@ static void get_active_binding(const struct sway_shortcut_state *state,
 			continue;
 		}
 
-		if (*current_binding && *current_binding != binding &&
-				strcmp((*current_binding)->input, binding->input) == 0) {
-			sway_log(SWAY_DEBUG, "encountered duplicate bindings %d and %d",
-					(*current_binding)->order, binding->order);
-		} else if (!*current_binding ||
-				strcmp((*current_binding)->input, "*") == 0) {
-			*current_binding = binding;
-
-			if (strcmp((*current_binding)->input, input) == 0) {
-				// If a binding is found for the exact input, quit searching
-				return;
+		if (*current_binding) {
+			if (*current_binding == binding) {
+				if ((*current_binding)->order != binding->order) {
```

Comment:
Hmm, I'm confused. How can this happen if `*current_binding == binding`?

Reply:
I just removed the check since it shouldn't happen. I think I read the previous condition wrong when refactoring